example/src/shared/utils.ts

```diff
@@ -1,3 +1,7 @@
+import { isEmpty } from 'lodash';
+
+const VIDEOROOM_URL = 'https://videoroom.membrane.work/room/';
```

Comment:
we're checking this url for deeplinking too, use it there too

and better yet put it in the .env files and get it from there

Reply:
Added env variable.